components/breadcrumbs/index.ts

```diff
@@ -0,0 +1 @@
+export { default as HeadlessBreadcrumbs } from "./src/HeadlessBreadcrumbs";
```

Comment:
Can we add another Component here. Which will have following features:
- Import HeadlessBreadCrumb
- Have a default seperatorr(>)
- Accepts an array of strings/ReactNode

Have a starting home Icon
We can call it ReusejsBreadcrumbs

Reply:
Also what is the behaviour of the number of `crumbs` are more. For example if we have 5 elements i the array. How will that be displayed.

Suggestions, If more elements than it is possible to be displayed in screen we have to hide elements and display .... in its place for eg:
Home > Link > ... > Sample

Please refer to the above example. This is a functional behavior so should be part of HeadlessBreadcrumb component.

Reply:
added ReusejsBreadcrums with default Home Icon and Separator ison also handle case for is there is more item in breadcrumbs then it can be managed with maxItems props.